tdrs-backend/tdpservice/users/management/commands/populate_stts.py

```diff
@@ -0,0 +1,80 @@
+"""`populate_stts` command."""
+
+import csv
+from pathlib import Path
+
+from django.core.management import BaseCommand
+
+from ...models import Region, STT
+
+
+DATA_DIR = BASE_DIR = Path(__file__).resolve().parent / "data"
+
+
+def _populate_regions():
+    with open(DATA_DIR / "regions.csv") as csvfile:
+        reader = csv.DictReader(csvfile)
+        for row in reader:
+            Region.objects.get_or_create(id=row["Id"])
+        Region.objects.get_or_create(id=1000)
+
+
+def _get_states():
+    with open(DATA_DIR / "states.csv") as csvfile:
+        reader = csv.DictReader(csvfile)
+        return [
+            STT(
+                code=row["Code"],
+                name=row["Name"],
+                region_id=row["Region"],
+                type=STT.STTType.STATE,
+            )
+            for row in reader
+        ]
+
+
+def _get_territories():
+    with open(DATA_DIR / "territories.csv") as csvfile:
+        reader = csv.DictReader(csvfile)
+        return [
+            STT(
+                code=row["Code"],
+                name=row["Name"],
+                region_id=row["Region"],
+                type=STT.STTType.TERRITORY,
+            )
+            for row in reader
+        ]
+
+
+def _populate_tribes():
+    with open(DATA_DIR / "tribes.csv") as csvfile:
+        reader = csv.DictReader(csvfile)
+        stts = [
+            STT(
+                name=row["Name"],
+                region_id=row["Region"],
+                state=STT.objects.get(code=row["Code"]),
+                type=STT.STTType.TRIBE,
+            )
+            for row in reader
+        ]
+        STT.objects.bulk_create(stts, ignore_conflicts=True)
+
+
+class Command(BaseCommand):
+    """Command class."""
+
+    help = "Populate regions, states, territories, and tribes."
+
+    def handle(self, *args, **options):
```

Comment:
@lauraGgit Logging is definitely a good idea, but because this script is run from the command line. We could import the bash username, which would be imperfect for obvious reasons, but even then when running in the dockerized container it would only return `root`. Thoughts?

Reply:
Logging has been added without the username

Reply:
I have added logging, but since this runs from the command line we can't really get the user. I could pull the user in with bash, but since it runs in docker the user will always be `root`.